ImageNet/inference/Intel_resnet50_c59xlarge_intelcaffe.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "Intel(R) Corporation",
+    "authorEmail": "daisy.deng@intel.com",
+    "framework": "Intel(R) Optimized Caffe",
+    "codeURL": "https://github.com/intel/caffe/tree/DAWN_Bench",
```

Comment:
@daisyden From the README for the repo, it isn't clear how you ran the experiment for the benchmark. What command or script did you run? Without that, I can't check that your implementation is correct. Please update the README.

Reply:
Hi, codyaustun
Do you mean I need to specify the test command and script in https://github.com/intel/caffe/tree/DAWN_Bench/README? I will add it and let you know. 
A question, when measuring the inference performance what is the expected way to load data? In this json file I uploaded we used compressed_lmdb and read the data from disk, can we cache the data into memory before testing?

Thanks
Daisy